docusaurus/docs/overview/feature-model/index.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: Data Model
+---
+
+## Key specifications
+
+Overture data is described by the [_simple feature model_](https://www.ogc.org/standard/sfa/) specified by the Open Geospatial Consortium. This means that each Overture _entity_ is represented as a single [GeoJSON](https://geojson.org/) feature: a geometric object with associated attributes and properties. For example, an _entity_ from the Overture buildings data theme might be represented in this way:
+
+```json
+{
+  "id": "08ba0085884a4fff0200ab2e3ad3f37c",
+  "type": "Feature",
+  "geometry": {
+    "type": "Polygon",
+    "coordinates": [
```

Comment:
@danabauer — I took out some of the newlines here to make it more readable; do you think this is OK? Do you think we could even go to just 1 line for the coordinates?

Reply:
looks good! No need to go to one line